pkg/ocp4/oauth/openid.go

```diff
@@ -26,8 +26,8 @@ type identityProviderOpenID struct {
 	} `yaml:"openID"`
 }
 
-func buildOpenIDIP(serializer *json.Serializer, p configv1.IdentityProvider) (identityProviderOpenID, secrets.Secret) {
-	var idP identityProviderOpenID
+func buildOpenIDIP(serializer *json.Serializer, p configv1.IdentityProvider) (IdentityProviderOpenID, secrets.Secret) {
```

Comment:
The casing of a function's argument is irrelevant to its accessibility, and as a matter of style, I think these should be lower cased.

Reply:
This can go in a different PR.

Reply:
Please file an issue then and link here before we merge this.